pkg/endpoint/bpf.go

```diff
@@ -179,15 +179,25 @@ func (e *Endpoint) addNewRedirectsFromDesiredPolicy(ingress bool, desiredRedirec
 			if !e.hasSidecarProxy || l4.L7Parser != policy.ParserTypeHTTP {
 				var finalizeFunc revert.FinalizeFunc
 				var revertFunc revert.RevertFunc
-				redirectPort, err, finalizeFunc, revertFunc = e.updateProxyRedirect(l4, proxyWaitGroup)
+
+				proxyID := ""
```

Comment:
It is equivalent to `var proxyID string`. It looked like a declaration is required due to `err` already being declared, but this can be simplified as:
```
@@ -172,7 +172,6 @@ func (e *Endpoint) addNewRedirectsFromDesiredPolicy(ingress bool, desiredRedirec
        for _, l4 := range m {
                if l4.IsRedirect() {
                        var redirectPort uint16
-                       var err error
                        // Only create a redirect if the proxy is NOT running in a sidecar
                        // container. If running in a sidecar container, just allow traffic
                        // to the port at L4 by setting the proxy port to 0.
@@ -180,8 +179,7 @@ func (e *Endpoint) addNewRedirectsFromDesiredPolicy(ingress bool, desiredRedirec
                                var finalizeFunc revert.FinalizeFunc
                                var revertFunc revert.RevertFunc
 
-                               proxyID := ""
-                               proxyID, err = e.ProxyID(e.desiredPolicy.NamedPortsMap, l4)
+                               proxyID, err := e.ProxyID(e.desiredPolicy.NamedPortsMap, l4)
                                if err != nil {
                                        // skip redirects for which a proxyID cannot be created.
                                        // Right now this happens only due to named port mapping not existing and
```

Thanks!

Reply:
Not addressed?